crates/ide/src/syntax_highlighting/tests.rs

```diff
@@ -143,12 +143,15 @@ mod inner {}
 
 pub mod ops {
```

Comment:
We can put these behind the minicore flags, no reason to define them in this test fixture again

Reply:
It's not used in the fixture, so I thought the goal was to highlight the definition.